tools/tune/dmic/dmic_init.m

```diff
@@ -55,17 +54,17 @@
 fprintf('Selected fifo_a_fs=%d, fifo_b_fs=%d: ', prm.fifo_a_fs, prm.fifo_b_fs);
 cfg = select_mode(common_mcic_list, a_mfir_list, b_mfir_list, common_clkdiv_list, cfg);
 cfg = get_cic_config(prm, cfg, hw);
-cfg = get_fir_config(prm, cfg, hw, spec);
+[cfg, prm] = get_fir_config(prm, cfg, hw, spec);
```

Comment:
`cfg` argument is not in use; if you need to return the `cfg` for any reason, please include a note. Thanks

Reply:
It remains from Matlab version of dmic_computed.c (in drivers directory). I think I can clean up the unused parts to avoid them to confuse.